helper/password.js

```diff
@@ -0,0 +1,18 @@
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+dotenv.config();
+export default class pawwordenclip {
+    static  enclippass(textPassword) {
+      const encpass = bcrypt.hashSync(textPassword,10);      
+     return encpass;
+    }
+    static verifypassword(inputPassword,storedPassword){
+        const boolPassword = bcrypt.compareSync(inputPassword,storedPassword);
+    return boolPassword;
+    }
+    static  jwtoken(generate) {
```

Comment:
can you put this in a separate file? this file is dealing with the password only. not the token. you can create a separate file that deals with the token